File: src/main/java/jukebot/commands/About.java
package jukebot.commands;

import com.sedmelluq.discord.lavaplayer.tools.PlayerLibrary;
import jukebot.JukeBot;
import jukebot.utils.Command;
import jukebot.utils.CommandProperties;
import jukebot.utils.Context;
import net.dv8tion.jda.core.JDAInfo;
import net.dv8tion.jda.core.entities.MessageEmbed;
import net.dv8tion.jda.core.entities.User;
import org.sqlite.SQLiteJDBCLoader;

@CommandProperties(aliases = {"info"}, description = "Displays some information about the bot")
public class About implements Command {

    @Override
    public void execute(final Context context) {
        final String dependencies = "**JDA**: " + JDAInfo.VERSION +
                "\n**Lavaplayer**: " + PlayerLibrary.VERSION +
                "\n**SQLite**: " + SQLiteJDBCLoader.getVersion();

        final User dev = context.getJda().getUserById(180093157554388993L);
        final MessageEmbed.Field[] fields = {
                new MessageEmbed.Field("Dependencies", dependencies, true),
                new MessageEmbed.Field("Links", "[GitHub](https://github.com/Devoxin/JukeBot)\n[Website](https://jukebot.xyz)", true)
        };

        context.sendEmbed("About JukeBot " + JukeBot.VERSION,
                "Developer: **" + dev.getName() + "#" + dev.getDiscriminator() + "**",
                fields);
    }
}

File: src/main/java/jukebot/commands/About.kt
package jukebot.commands

import com.sedmelluq.discord.lavaplayer.tools.PlayerLibrary
import jukebot.JukeBot
import jukebot.framework.Command
import jukebot.framework.CommandProperties
import jukebot.framework.Context
import jukebot.utils.addFields
import net.dv8tion.jda.api.JDAInfo
import net.dv8tion.jda.api.entities.MessageEmbed
import org.sqlite.SQLiteJDBCLoader

@CommandProperties(aliases = ["info"], description = "Displays some information about the bot")
class About : Command(ExecutionType.STANDARD) {

    override fun execute(context: Context) {
        val dependencies = "**JDA**: ${JDAInfo.VERSION}" +
            "\n**Lavaplayer**: ${PlayerLibrary.VERSION}" +
            "\n**SQLite**: ${SQLiteJDBCLoader.getVersion()}"

        val fields = arrayOf(
            MessageEmbed.Field("Dependencies", dependencies, true),
            MessageEmbed.Field("Links", "[GitHub](https://github.com/Devoxin/JukeBot)\n[Website](https://jukebot.serux.pro)", true)
        )

        context.embed {
            setTitle("JukeBot ${JukeBot.VERSION}")
            setDescription("Developer: **devoxin#0101**")
            addFields(fields)
        }
    }
}
